js/app/directives/slidetoggleDirective.js

```diff
@@ -0,0 +1,74 @@
+/*
+
+@author Bernhard Posselt
+@copyright 2012 Bernhard Posselt dev@bernhard-posselt.com
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+License as published by the Free Software Foundation; either
+version 3 of the License, or any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+app.directive('ocClickSlideToggle', [
```

Comment:
IIRC this is more or less in core now ;)

See data-apps-slide-toggle: https://doc.owncloud.org/server/8.1/developer_manual/app/css.html#settings-area